Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -17,7 +30,7 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate {
     lazy var payPalClient: PayPalWebCheckoutClient = {
         let clientID = DemoSettings.clientID
         let environment = DemoSettings.environment.paypalSDKEnvironment
-        let config = CoreConfig(clientID: clientID, environment: environment)
+        let config = CoreConfig(clientID: clientID, secret: DemoSettings.secret, environment: environment)
```

Comment:
qq: Do we still need this? It's still in the Android app too but I couldn't remember why we need it. Eventually the client secret should be phased out.

Reply:
@sshropshire yeap, `confirm_payment_source` doesnt accept low scope access token yet.